zone-service/src/main/java/org/cloudfoundry/identity/uaa/zone/model/ConnectionDetails.java

```diff
@@ -0,0 +1,12 @@
+package org.cloudfoundry.identity.uaa.zone.model;
+
+import lombok.Data;
+
+@Data
+public class ConnectionDetails {
```

Comment:
Not sure how, but you used all these model classes in previous commit but creating them here in this commit?

Reply:
Rebased and ordered the commit.